cfgov/legacy/static/knowledgebase/less/es-ask-base.less

```diff
@@ -712,6 +712,24 @@ legend {
   }
 }
 
+/* ==========================================================================
+   Wagtail Ask CFPB styles
+   ========================================================================== */
+  
+.responsive-object {
```

Comment:
Scoped in https://github.com/cfpb/cfgov-refresh/pull/3059/commits/e45f29fda9882cbf1f6f643b72d5f80b427d8f77.